tests/e2e-leg-6/revivedb-1-node-restore/26-verify-first-restore.yaml

```diff
@@ -0,0 +1,30 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # - script: |-
```

Comment:
Lets open a jira and include it in the comments so that we know if/when this becomes stale.

Reply:
will do

Reply:
ticket number added in the comment